endaq/calc/utils.py

```diff
@@ -206,3 +211,49 @@ def _rolling_slice_definitions(
     num = int(slice_width / spacing / 2)
 
     return indexes, slice_width, num, length
+
+
+def convert_units(
+        src: str,
+        dst: str,
+        df: pd.DataFrame = None,
+        print_conversion: bool = False,
```

Comment:
This option to print seems a little weird; is it needed? For the most part, functions in libraries rarely mix printing and returning things.